### PR TITLE
Update to PHP 7.1.18, Uuid 3.7.3 & Guzzle 6.3.3

### PR DESCRIPTION
**Pull request by akrabat**
- Update PHP to 7.1.18
- Update ramsey/uuid to 3.7.3
- Update guzzlehttp/guzzle to 6.3.3
- Add a CHANGELOG
- Remove empty continuation lines in `Dockerfile` for future proofing.

### CODE REVIEW

File: core/php7.1Action/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 1.0.1
+
+- Change: Update PHP to 7.1.8
```
Comment (rabbah) [inline core/php7.1Action/CHANGELOG.md]: this seems wrong since 1.0.0 says 7.1.9

Comment (rabbah) [reply core/php7.1Action/CHANGELOG.md]: should be 7.1.18 matching the dockerfile.

File: core/php7.1Action/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Change: Update [guzzlehttp/guzzle](https://packagist.org/packages/guzzlehttp/guzzle) to 6.3.3
+- Change: Update [ramsey/uuid](https://packagist.org/packages/ramsey/uuid) to 3.7.3
+
+## 1.0.0
+Initial release
+
+- Added: PHP: 7.1.9.
```
Comment (rabbah) [inline core/php7.1Action/CHANGELOG.md]: was this 7.1.9?

Comment (akrabat) [reply core/php7.1Action/CHANGELOG.md]: > was this 7.1.9?

I believe so. At least, that's what `openwhisk/action-php-v7.1:latest` from Docker Hub has.

**Comment by akrabat** [comment]
Fixed that version number @rabbah. Do you want me to rebase it into the previous commit?

**Comment by rabbah** [comment]
not necessary should be able to rebase and merge as there are no conflicts.

**Pull request merged**